docs/managing-program-users.md

```diff
@@ -3,4 +3,14 @@ id: managing-program-users
 title: Managing Program Users
 ---
 
-Placeholder
+If you are a Program Administrator, then you will have the ability to invite collaborators to access your programs data.
+
+## How to Add Users
+1. Navigate to the **Submission** area in the top menu.
+1. Click **Manage Programs** in the .
+1. Above the users table, click the *Add Users* button.
+1. Fill in the First Name, Last Name, Email, and role of the person you would like to collaborate on your ARGO program.
+> It is very important that you use a Gmail or Gsuite enabled email address to invite any ARGO users.
```

Comment:
Should we add:
"This will be the email that they will log into the ICGC ARGO platform with."